src/probnum/statespace/integrator.py

```diff
@@ -201,11 +204,15 @@ def _dispmat(self):
     def equivalent_discretisation_preconditioned(self):
         """Discretised IN THE PRECONDITIONED SPACE."""
         empty_shift = np.zeros(self.spatialdim * (self.ordint + 1))
+        process_noise_cov_cholesky = np.linalg.cholesky(
+            self._proc_noise_cov_mat
+            + self._process_noise_damping * np.eye(len(self._proc_noise_cov_mat))
+        )
```

Comment:
Is this nugget necessary for numerical stability?

Reply:
only for orders > 11 :)

Reply:
Should we then actually add this argument? I guess it will be needed quite rarely, if not only for the test you added in this PR 😄 Correct me if this is wrong

Reply:
Can be ignored, I read your answer below

Reply:
no, it is only for a bunch of extreme cases (would we be the only ODE solver of order 15 in python then :thinking:?) I will add a sentence into the docstring.

Reply:
no, not really. I was playing around with it and thought it was cool, but I see your concerns. Shall I remove it?